drs_downloader_to_getm.wdl

```diff
@@ -177,6 +185,13 @@ task download {
             done
```

Comment:
Why the `-P` option for `curl`?
According to `curl --help` on Ubuntu 18.04 LTS, it means:
```
 -P, --ftp-port <address> Use PORT instead of PASV
 ```
 
 For `wget`, the `-P` options means prefix, which makes sense, yet it doesn't appear to do the same thing for curl(?)

Reply:
Sigh, sorry about that.  Should be resolved now.